starter-code/src/viking.js

```diff
@@ -1,11 +1,119 @@
 // Soldier
-function Soldier() {}
+function Soldier(health, strength) {
+    this.health = health;
+    this.strength = strength;
+
+}
+
+//Una función para muchas instancias
+Soldier.prototype.attack = function(){
+ return this.strength;
+}
+
+Soldier.prototype.receiveDamage = function(damage){
+  this.health = this.health - damage;
+}
 
 // Viking
-function Viking() {}
 
+Viking.prototype = Object.create(Soldier.prototype);
+Viking.prototype.constructor = Viking;
+
+function Viking(name, health, strength) {
+    Soldier.call(this, health, strength);
```

Comment:
Para las herencias de las propiedades solo hay q declarar en el soldier.call 

function Viking(name, health, strength) {
    this.name = name;
    Soldier.call(this, health, strength);
}